README.md

```diff
@@ -15,25 +15,53 @@
 
-* Simple and direct integration with the [VS Code extension API](https://code.visualstudio.com/api/language-extensions/overview)
-* Implemented in [TypeScript](https://www.typescriptlang.org/), runs in [NodeJS](https://nodejs.org/)
-* Generates an abstract syntax tree from a grammar declaration
-* The parser is based on [Chevrotain](https://chevrotain.io/)
-* Grammar declaration language similar to [Xtext](https://www.eclipse.org/Xtext/documentation/301_grammarlanguage.html)
-* High out-of-the-box functionality, high extensibility
+* **Semantics First:** Building on top of a [grammar declaration language](https://langium.org/docs/grammar-language/), Langium enables you to build the abstract model of your language in parallel to its syntax. Langium parsers are powered by [Chevrotain](https://chevrotain.io).
+* **Lean by Default, Customizable by Design:** Langium offers the infrastructure you need to build languages purely by defining their grammar. If that is not enough, you can fine tune every detail of your language using our [dependency injection system](https://langium.org/docs/configuration-services/).
+* **Write Once, Run Everywhere:** By leveraging the flexibility of JavaScript and the [Language Server Protocol](https://microsoft.github.io/language-server-protocol/), a language written using Langium can run in all kind of IDEs, browsers or embedded in CLIs and server applications.
 
-## Why?
+[Learn how to use Langium in your project](https://langium.org/docs/getting-started/).
 
-A domain-specific language (DSL) enables domain experts to contribute automatically processable content without any programming knowledge. The contributed content can describe anything like structural data, behavior, or a mixture of both. By plugging in code generators that operate on the DSL, you create a powerful tool chain that creates technical artifacts from high-level specifications, improves communication between engineers as well as non-technical stakeholders and boosts the overall efficiency.
+## Installation
 
-The main goal of Langium is to lower the barrier of creating a DSL / low-code platform. We achieve this by providing a DSL that describes the syntax and structure of your language: the _grammar language_. Langium's out-of-the-box functionality is based on the information extracted from a grammar declaration. The Xtext framework has proved this approach to be suitable both for rapid prototyping and large-scale applications.
+Build your first language with Langium in our [online playground](https://langium.org/playground/).
 
-## How?
+Alternatively you can use yeoman to generate a [sample Langium project](https://langium.org/docs/getting-started/).
```

Comment:
I wouldn't call this an alternative, but rather the recommended approach once you're past the "playing around" phase and want to set up a project.